app/src/main/java/org/brightmindenrichment/street_care/ui/user/ChangePasswordFragment.kt

```diff
@@ -0,0 +1,118 @@
+package org.brightmindenrichment.street_care.ui.user
+
+import android.content.Intent
+import android.os.Bundle
+import android.provider.MediaStore
+import android.text.TextUtils
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.EditText
+import android.widget.Toast
+import androidx.activity.result.ActivityResultLauncher
+import androidx.activity.result.contract.ActivityResultContracts
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import com.google.android.material.bottomnavigation.BottomNavigationView
+import com.google.android.material.button.MaterialButton
+import com.google.firebase.auth.EmailAuthProvider
+import com.google.firebase.auth.FirebaseAuth
+import com.google.firebase.auth.ktx.auth
+import com.google.firebase.ktx.Firebase
+import org.brightmindenrichment.street_care.R
+import org.brightmindenrichment.street_care.databinding.ChangePasswordBinding
+import org.brightmindenrichment.street_care.util.Extensions
+
+class ChangePasswordFragment : Fragment() {
+    private var _binding: ChangePasswordBinding? = null
+    private val binding get() = _binding!!
+    private lateinit var auth: FirebaseAuth
```

Comment:
these `var` declarations can be localized inside their respective function usage

```
    private fun onSaveChanges() {
        val oldpassword = binding.editTextOldPassword.text.toString()
        val newpassword = binding.editTextNewPassword.text.toString()
        val reenternewpassword = binding.editTextReenterNewPassword.text.toString()